core/common/src/main/java/alluxio/underfs/BaseUnderFileSystem.java

```diff
@@ -62,7 +62,7 @@ public void configureProperties() throws IOException {
 
   @Override
   public OutputStream create(String path) throws IOException {
-    return create(path, CreateOptions.defaults());
+    return create(path, CreateOptions.defaults().setCreateParent(true));
```

Comment:
It seems like we would need to split out the journal interface from the ufs interface if we want to address this.